cli/pkg/kctrl/cmd/app/release/release.go

```diff
@@ -22,11 +22,11 @@ type ReleaseOptions struct {
 	chdir          string
 	outputLocation string
 	debug          bool
+	tag            string
```

Comment:
nit: Should we call the variable `bundleTag` for the time being as that is the only thing it is used for?

Reply:
Adopted